client-sdk/go/modules/contracts/types.go

```diff
@@ -12,6 +14,13 @@ type CodeID uint64
 // InstanceID is the unique deployed code instance identifier.
 type InstanceID uint64
 
+// Address returns address for the InstanceID.
+func (i *InstanceID) Address() types.Address {
+	id := make([]byte, 8)
+	binary.BigEndian.PutUint64(id, uint64(*i))
+	return types.NewAddressForModule("contracts", id)
+}
```

Comment:
Would be good to have a test case to make sure it is equal to the Rust side.

Reply:
It was implicitly tested via `NewAddressForModule` tests before. Added explicit tests for `InstanceId`->`Address` now to both rust and go sides.